frontend/app/utils/string-utils.ts

```diff
@@ -5,3 +5,13 @@ export function randomString(len: number, allowedChars = '0123456789abcdefghijkl
   const toRandomChar = () => allowedChars[Math.floor(Math.random() * allowedChars.length)];
   return Array(len).fill(undefined).map(toRandomChar).join('');
 }
+
+/**
+ * @param value - The whole number percentage (i.e. multiplied by 100) to be formatted
+ * @param locale - The Canadian locale to be used for formatting
+ * @returns - The number formatted as a percentage in the givenlocale
+ */
+export function formatPercent(value: number, locale: Language) {
+  if (!/en|fr/i.test(locale)) throw new Error(`locale must be 'en' or 'fr'`);
```

Comment:
`locale` will always be `en` or `fr` because of https://github.com/DTS-STN/future-sir/blob/main/frontend/app/%40types/global.d.ts#L19

Also, I think `locale` should be renamed to match `language` to match the type.

Reply:
"`always`" if you don't pass `any` as `locale` argument or call it in Javascript where Typescript doesn't matter.

Reply:
Sure. If you want to be pedantic. Maybe we also check that `value` isn't a URL or a JSX.Element.

For all intents and purposes, the check can be removed.

Reply:
Also.. @Dario-Au, can you add the return type to the function? We've been trying to be explicit about that to enhance readability.